components/custom-radio-button/design.md

```diff
@@ -0,0 +1,7 @@
+<!-- @license CC0-1.0 -->
+
+<!-- markdownlint-disable first-line-h1 -->
+
+## Design
+
+[Figma - Utrecht Design System - Custom RadioButton](https://www.figma.com/file/msb3CfQBefPoruqNQ968Zh/Utrecht-Design-System?node-id=499%3A5622)
```

Comment:
This links to the Figma frame for all Inputs, and I think it would be helpful to link directly to the Radio button frame.